modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -13,21 +13,21 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 * You have completed the AWS prerequisites for {hcp-title}.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
-+
-[NOTE]
-====
-To successfully install {hcp-title} clusters, use the latest version of the ROSA CLI (`rosa`).
-====
+* You have installed and configured ROSA CLI (`rosa`) version 1.2.23 or later on your installation host.
```

Comment:
We have enabled creating HCP account roles with managed policies in release `1.2.26`, before that it was blocked in the production environment.
```suggestion
* You have installed and configured ROSA CLI (`rosa`) version 1.2.26 or later on your installation host.
```